cmd/node/node.go

```diff
@@ -95,62 +95,74 @@ const (
 	ConStateLogger         = "conState"
 )
 
-// Cmd is the cobra wrapper for the node, that allows adding parameters to it.
-var Cmd = &cobra.Command{
-	Use:   "node",
-	Short: "start node",
-	Run: func(cmd *cobra.Command, args []string) {
-		conf, err := loadConfig(cmd)
-		if err != nil {
-			log.With().Fatal("failed to initialize config", log.Err(err))
-		}
+func GetCommand() *cobra.Command {
```

Comment:
I think if one calls `GetCommand()` mulitple times this might cause issues, since there is a side effect on line 138 with

```golang
	cmdp.AddCommands(cmd)
```

Can you wrap this into a `sync.Once` to make sure that multiple calls to `GetCommand()` always return the same instance (and still be concurrency safe?)

Reply:
What is the side effect? This call adds flags to local `cmd`.

Reply:
You're right; I was confused by the name `AddCommands` and assumed that `cmd` would be added to something. A better name for the function would probably be `ParseCmdFlags` or similar. But yes `sync.Once` isn't needed here.